cli/run.go

```diff
@@ -46,6 +46,7 @@ var runCommand = cli.Command{
 		// cli params used in CI mode
 		cli.StringFlag{Name: JSONParamsKey, Usage: "Specify command flags with json string-string hash."},
 		cli.StringFlag{Name: JSONParamsBase64Key, Usage: "Specify command flags with base64 encoded json string-string hash."},
+		cli.StringFlag{Name: OutputFormatKey, Usage: "test output"},
```

Comment:
Should we add a better usage description here? I would even say that this json format is mostly for internal usage and what if we just supply an empty string as the usage description?

Reply:
I think we can provide a valid description for the flag here. Not likely, but others can use this feature too.